### PR TITLE
WLA-627: Show translated Yes/No on CYA

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
https://tools.hmcts.net/jira/browse/WLA-627

### CODE REVIEW

File: steps/appointee/same-address/SameAddress.js [hunk 1]
```diff
@@ -23,10 +24,12 @@ class SameAddress extends SaveToDraftStore {
   }
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
+
     return answer(this, {
       question: this.content.cya.isAddressSameAsAppointee.question,
       section: sections.appellantDetails,
-      answer: titleise(this.fields.isAddressSameAsAppointee.value)
+      answer: titleise(content.cya.isAddressSameAsAppointee[this.fields.isAddressSameAsAppointee.value])
```
Comment (srochani) [inline steps/appointee/same-address/SameAddress.js:32]: What about content.en? it seems only done for cya

Comment (LucaDelBuonoHMCTS) [reply steps/appointee/same-address/SameAddress.js:32]: Not sure what you mean

File: steps/hearing/the-hearing/TheHearing.js [hunk 2]
```diff
@@ -24,11 +25,13 @@ class TheHearing extends SaveToDraftStore {
...
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
+
     return [
       answer(this, {
         question: this.content.cya.attendHearing.question,
         section: sections.theHearing,
-        answer: titleise(this.fields.attendHearing.value)
+        answer: titleise(content.cya.attendHearing[this.fields.attendHearing.value])
```
Comment (srochani) [inline steps/hearing/the-hearing/TheHearing.js:34]: same here

File: steps/identity/appointee/Appointee.js [hunk 3]
```diff
@@ -26,10 +27,12 @@ class Appointee extends SaveToDraftStore {
   }
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
+
     return answer(this, {
       question: this.content.cya.isAppointee.question,
       section: sections.appointeeDetails,
-      answer: titleise(this.fields.isAppointee.value)
+      answer: titleise(content.cya.isAppointee[this.fields.isAppointee.value])
```
Comment (srochani) [inline steps/identity/appointee/Appointee.js:35]: same here

File: steps/appointee/same-address/SameAddress.js (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply steps/appointee/same-address/SameAddress.js:32]: Oh I think I understand what you're saying.
It's CYA (Check Your Appeal) - not CY (Welsh).
As you can see, the correct `content` (EN or CY) gets loaded dynamically on line 27.
Then in line 32 it takes the `.cya.isAddressSameAsAppoint` property from it and matches it to what the user answered (`yes` or `no`).

Comment (srochani) [reply steps/appointee/same-address/SameAddress.js:32]: ah ok, its confusing content.cya and content.cy, can we please not change to relevant

File: steps/appointee/same-address/SameAddress.js [hunk 4]
```diff
@@ -23,10 +24,12 @@ class SameAddress extends SaveToDraftStore {
   }
 
   answers() {
+    const content = require(`./content.${i18next.language}`);
```
Comment (srochani) [inline steps/appointee/same-address/SameAddress.js:27]: Can you please move this line outside of the method

Comment (LucaDelBuonoHMCTS) [reply steps/appointee/same-address/SameAddress.js:27]: That wouldn't work as `i18next` is still not initialised at that point.

**Pull request merged**